[2024-08-07T14:33:30.879+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T14:33:31.057+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:33:31.076+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [queued]>
[2024-08-07T14:33:31.094+0000] {taskinstance.py:2306} INFO - Starting attempt 4 of 4
[2024-08-07T14:33:31.151+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): Autorizacion_NASA> on 2024-08-07 14:20:08.695612+00:00
[2024-08-07T14:33:31.165+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=775) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T14:33:31.168+0000] {standard_task_runner.py:64} INFO - Started process 784 to run task
[2024-08-07T14:33:31.168+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***-space-exploration', 'Autorizacion_NASA', 'manual__2024-08-07T14:20:08.695612+00:00', '--job-id', '126', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmpwwn3znme']
[2024-08-07T14:33:31.171+0000] {standard_task_runner.py:91} INFO - Job 126: Subtask Autorizacion_NASA
[2024-08-07T14:33:31.257+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow-space-exploration.Autorizacion_NASA manual__2024-08-07T14:20:08.695612+00:00 [running]> on host 38873943cc86
[2024-08-07T14:33:31.417+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-space-exploration' AIRFLOW_CTX_TASK_ID='Autorizacion_NASA' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T14:20:08.695612+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-07T14:20:08.695612+00:00'
[2024-08-07T14:33:31.418+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T14:33:31.440+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T14:33:31.441+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "sleep 20 && echo 'OK' > /tmp/response_20240807.txt"]
[2024-08-07T14:33:31.457+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T14:33:52.494+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T14:33:52.495+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T14:33:52.527+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***-space-exploration, task_id=Autorizacion_NASA, run_id=manual__2024-08-07T14:20:08.695612+00:00, execution_date=20240807T142008, start_date=20240807T143331, end_date=20240807T143352
[2024-08-07T14:33:52.568+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T14:33:52.597+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T14:33:52.599+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
